Linux/Car_Client/chronos.cpp

```diff
@@ -35,6 +35,8 @@ Chronos::Chronos(QObject *parent) : QObject(parent)
             this, SLOT(processOpro(chronos_opro)));
     connect(mChronos, SIGNAL(rcmmRx(chronos_rcmm)),
             this, SLOT(processRcmm(chronos_rcmm)));
+	connect(mChronos, SIGNAL(heabTimeOut()),
```

Comment:
This was done due to differences in QtElapsedTimer and QtTimer, the latter works by repeatedly emitting a timeout signal after a given interval, in our case this triggers the checkHeabTimer function.

The former instead returns the elapsed time since started/restarted allowing us to use that value to check against the define. 

As you said, using one timer would be nicer